docs-src/content/protobuf/_index.md

```diff
@@ -29,8 +29,9 @@ Service that implements the Cobalt Diatheke Dialog Management API.
 | CreateSession | [SessionStart](#SessionStart) | [SessionOutput](#SessionOutput) | Create a new Diatheke session. Also returns a list of actions to take next. |
 | DeleteSession | [TokenData](#TokenData) | [Empty](#Empty) | Delete the session. Behavior is undefined if the given TokenData is used again after this function is called. |
 | UpdateSession | [SessionInput](#SessionInput) | [SessionOutput](#SessionOutput) | Process input for a session and get an updated session with a list of actions to take next. This is the only method that modifies the Diatheke session state. |
-| StreamASR | [ASRInput](#ASRInput) | [ASRResult](#ASRResult) | Create an ASR stream. A result is returned when the stream is closed by the client (which forces the ASR to endpoint), or when a transcript becomes available on its own, in which case the stream is closed by the server. The ASR result may be used in the UpdateSession method. |
+| StreamASR | [ASRInput](#ASRInput) | [ASRResult](#ASRResult) | Create an ASR stream. A result is returned when the stream is closed by the client (which forces the ASR to endpoint), or when a transcript becomes available on its own, in which case the stream is closed by the server. The ASR result may be used in the UpdateSession method. <br/><br/> If the session has a wakeword enabled, and the client application is using Diatheke and Cubic to handle the wakeword processing, this method will not return a result until the wakeword condition has been satisfied. Utterances without the required wakeword will be discarded and no transcription will be returned. |
 | StreamTTS | [ReplyAction](#ReplyAction) | [TTSAudio](#TTSAudio) | Create a TTS stream to receive audio for the given reply. The stream will close when TTS is finished. The client may also close the stream early to cancel the speech synthesis. |
+| Transcribe | [TranscribeInput](#TranscribeInput) | [TranscribeResult](#TranscribeResult) | Create an ASR stream for transcription. Unlike StreamASR, Transcribe does not listen for a wakeword. This method returns a bi-directional stream, and its intended use is for situations where a user may say anything at all, whether it is short or long, and the application wants to save the transcript (e.g., take a note, send a message). <br/><br/> The first message sent to the server must include the Cubic model ID, with remaining messages sending audio data. Messages received from the server will include the current best partial transcription until the full transcription is ready. The stream ends when either the client application closes it, a predefined duration of silence (non-speech) occurs, or the end-transcription intent is recognized. |
```

Comment:
Does the first message include the *Cubic* model id or the *Diatheke* model id?  I thought the Cubic model for transcription was set in the Diatheke config file.

Reply:
FYI, this was auto-generated from the protobuf file.

In the server's current implementation, it is expecting the Cubic model ID to be there, so it is correct as written. The fact that there is an associate created between a Diatheke model and a Cubic model in the server config file is significant for generating a TranscribeAction (which the server will populate with both the Cubic and Diatheke model ID), but not required for this stream. This allows a client application to create a Transcribe stream with just a Cubic model ID at any time they want (maybe in response to a button click, for example), without it being prompted by a TranscribeAction in the session output. They could even add the Diatheke model ID too, to get the stream-end conditions if they want it, and it doesn't have to be the same association as defined in the server config file. Think of it more like this:

* Cubic Model ID = Always used for ASR, regardless of optional Diatheke model.
* Diatheke Model ID = Optional, but always used to to define the end-stream conditions.